src/main/resources/templates/products/show.html

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
Please remove extra line. 
Also between `<link...` and `</head`
as well as the `<h1>` and the `<div>`